### PR TITLE
Fixes for Crowd blog

### PR DESCRIPTION
**Pull request by zalapeach**
## This pull request involves

* Fix order of previous/next links
* Fix layout for bread cums in blogpost entry
* Remove "we talk about" in mobile and change for "we write about" in desktop

## Description

There are some little changes for the Crowd's blog

### CODE REVIEW

File: app/assets/stylesheets/modules/menu-header.scss [hunk 1]
```diff
@@ -2,9 +2,24 @@
   margin: 0 auto;
   padding: 25px 5% 0;
   text-align: center;
+  @include mq-tablet() {
+    max-width: 768px;
+    padding: 25px 20px 0;
+    width: 95%;
+  }
+  @include mq-mobile() {
+    padding: 25px 5% 0;
```
Comment (rafaelramos) [inline app/assets/stylesheets/modules/menu-header.scss]: Try to order this alphabetically

File: app/assets/stylesheets/modules/menu-header.scss [hunk 2]
```diff
@@ -13,10 +28,32 @@
...
   a {
     color: $gray-medium-lighter;
-    font-weight: bold;
     text-decoration: none;
   }
 
-  .previous { float: right; }
-  .next { float: left; }
+  .category-link {
+    position: absolute;
```
Comment (rafaelramos) [inline app/assets/stylesheets/modules/menu-header.scss]: also here, try to order them alphabetically

**Comment by zalapeach** [comment]
Done and thanks @rafaelramos :)

**Comment by jaime-evr** [comment]
LGTM

**Pull request merged**